system/services/siteTree/SiteTreeService.cfc

```diff
@@ -1357,17 +1357,32 @@ component {
 		return page.id ?: "";
 	}
 
-	public void function clearPageCache( required string pageId ) {
-		var pageUrl    = ReReplace( $getRequestContext().buildLink( page=arguments.pageId ), "^https?://.*?/", "/" );
+	public void function clearAllCaches() {
+		_getCachebox().clearAll();
+		announceInterception( "onClearCaches", {} );
```

Comment:
This needs `$` in front of `$announceInterception()`?

Reply:
Arh yes, nice catch